docs/basics/networkvariable.md

```diff
@@ -25,13 +26,16 @@ You can use `NetworkVariable` [permissions](#permissions) to control read and wr
 - A `NetworkVariable`'s value can only be set:
     - When initializing the property (either when it's declared or within the Awake method).
     - While the associated NetworkObject is spawned (upon being spawned or any time while it's still spawned).
+- A `NetworkVariable` cannot be defined with the `static` keyword. Variables defined with the static keyword have a lifetime that lasts for the entire runtime of an application, rather than the much shorter lifetime of a `GameObject`. This results in incorrect and invalid state management where a `NetworkVariable` will not behave as expected.
 
-###  Initializing a NetworkVariable
+### Initializing a NetworkVariable
 
-When a client first connects, it's synchronized with the current value of the `NetworkVariable`. Typically, clients should register for `NetworkVariable.OnValueChanged` within the `OnNetworkSpawn` method. A `NetworkBehaviour`'s `Start` and `OnNetworkSpawn` methods are invoked based on the type of NetworkObject the `NetworkBehaviour` is associated with:   
+When the associated NetworkObject of a `NetworkBehaviour` with `NetworkVariable` properties is spawned, the associated `NetworkVariable` will also be spawned.The `NetworkVariable`'s current state (`Value`) is automatically synchronized on the client side. When any new client connects, it's synchronized with the current value of the `NetworkVariable` before the `NetworkBehaviour.OnNetworkSpawn` is invoked.
```

Comment:
> the associated `NetworkVariable` will also be spawned. The `NetworkVariable`'s current state (`Value`) is automatically synchronized on the client side.

Perhaps:
the associated `NetworkVariable` will be initialized and associated with its associated `NetworkBehaviour`. 

Remove:

> The `NetworkVariable`'s current state (`Value`) is automatically synchronized on the client side.

This could be confusing since in a client-server network topology it will be the server spawning and doing the initial local "spawning"/synchronization of the value.